paymentsheet-example/src/androidTest/java/com/stripe/android/test/core/PlaygroundTestDriver.kt

```diff
@@ -826,10 +828,14 @@ internal class PlaygroundTestDriver(
                         .size == 1
                 }
 
-                composeTestRule.waitUntil {
-                    composeTestRule.onAllNodesWithText("Country or region")
-                        .fetchSemanticsNodes()
-                        .size == 1
+                val collectionMode = testParameters.playgroundSettingsSnapshot[CollectAddressSettingsDefinition]
+
+                if (collectionMode != PaymentSheet.BillingDetailsCollectionConfiguration.AddressCollectionMode.Never) {
+                    composeTestRule.waitUntil {
+                        composeTestRule.onAllNodesWithText("Country or region")
+                            .fetchSemanticsNodes()
+                            .size == 1
+                    }
```

Comment:
The `Link` tests cut off the top logos. I'm choosing to hide the billing address for the `Link` tests so that they can be seen.